ait/commons/util/command/upload.py

```diff
@@ -54,73 +105,37 @@ def get_files(upload_path, curr_path, level):
                         # skip hidden files and dirs
                         if not exclude(f):
                             if os.path.isfile(full_path):
-                                f_size = os.path.getsize(full_path)
-                                rel_path = full_path.replace(upload_path + ('' if upload_path.endswith('/') else '/'), '')
-                                fs.append(FileTransfer(path=full_path, key=rel_path, size=f_size))
+                                files.append(full_path)
 
                             elif os.path.isdir(full_path):
                                 get_files(upload_path, full_path, level)
 
             for p in ps:
```

Comment:
Please use a more meaningful name. You will thank yourself when you come back to this code in a few weeks.